src/compiler/crystal/codegen/primitives.cr

```diff
@@ -135,7 +135,7 @@ class Crystal::CodeGenVisitor
       return codegen_binary_op_with_overflow(op, t1, t2, p1, p2)
     end
 
-    tmax, p1, p2 = codegen_binary_extend_int(t1, t2, p1, p2)
+    _, p1, p2 = codegen_binary_extend_int(t1, t2, p1, p2)
```

Comment:
Everywhere a name turns into an underscore it makes the code more cryptic. Can we change these (there are several) into `_tmax` and so on? They are unused, but they give context of what is being returned, and what's being unpacked.

Reply:
Fixed